### DISCUSSION TITLE
Rename Volume Missing?

### DISCUSSION THREAD

**Discussion by jr0dsgarage** [fadden/CiderPress2#12] [General]
I was going to make a bug report, but am not sure that I'm not just missing this somewhere in the new UI?

How do I rename a volume in CP2?

**Comment by fadden** [comment]
Select the volume directory in the Directories panel, then Actions > Edit Directory Attributes.  Also works for changing the volume number in a DOS VTOC.

I'm not entirely happy with the way this works.  Edit Directory Attributes always edits the thing that is selected in Directories, which feels weird if you have a different directory selected in the file list.  I expect most people will use the right-click context menus rather than the top menu, so it's probably okay.  An alternative would be to have two commands called Edit Attributes, one for the file list, one for the directory tree, but then it's unclear what Actions > Edit Attributes should do (don't get me started on tracking focus in WPF).

There could also be a dedicated Actions > Edit Volume Attributes, but that seems like clutter.

It's Edit Attributes rather than Rename because ProDOS and HFS have dates you can edit.  I find myself looking for a Rename option sometimes.

(Alternatively: `cp2 mv disk.po : NewName`.)

See also https://ciderpress2.com/gui-manual/editattrs.html, near the bottom.

**Comment by jr0dsgarage** [comment]
Yeah, I guess I'm just used to looking for the word "Volume" to rename the root prefix of the disk.

**Comment by jr0dsgarage** [comment]
I realized I asked here first before checking the manual I totally know about.  closing this discussion!
